prowler/providers/azure/exceptions/exceptions.py

```diff
@@ -0,0 +1,162 @@
+from prowler.exceptions.exceptions import ProwlerException
+
+
+class AzureBaseException(ProwlerException):
+    """Base class for Azure Errors."""
+
+    AZURE_ERROR_CODES = {
+        (1914, "AzureEnvironmentVariableError"): {
+            "message": "Azure environment variable error",
+            "remediation": "Check the Azure environment variables and ensure they are properly set.",
+            "file": "{file}",
+            "provider": "Azure",
```

Comment:
Why do we need to set `"provider": "Azure"` if these are just for the Azure provider?

Reply:
True, I will change this.